lib/internal/bootstrap/node.js

```diff
@@ -244,7 +244,8 @@ Object.defineProperty(process, 'features', {
     tls_alpn: hasOpenSSL,
     tls_sni: hasOpenSSL,
     tls_ocsp: hasOpenSSL,
-    tls: hasOpenSSL
+    tls: hasOpenSSL,
+    code_cache: config.hasCodeCache,
```

Comment:
Maybe a better key is `embedded_code_cache`, or `builtin_code_cache`, since we do support user land code cache in `vm` module separately.

Reply:
Or if we want to be more succinct...`cached_builtin`?

Reply:
Went with `cached_builtins`.